chia/full_node/mempool_manager.py

```diff
@@ -722,8 +860,17 @@ def can_replace(
         # fee than AB therefore kicking out A altogether. The better way to solve this would be to keep a cache
         # of booted transactions like A, and retry them after they get removed from mempool due to a conflict.
         for coin in item.removals:
-            if coin.name() not in removal_names:
-                log.debug(f"Rejecting conflicting tx as it does not spend conflicting coin {coin.name()}")
+            coin_id = coin.name()
+            if coin_id not in removal_names:
+                # NOTE: Coins that are eligible for deduplication are excluded from the superset rule. This
+                # prevents an attacker from replacing a transaction with one that spends an eligible coin
+                # additionally, rendering it impossible to replace the original transaction by fee later on.
```

Comment:
I don't think this comment is correct. It is possible to replace such transaction, it's just that our wallet doesn't support it. But it doesn't support replace-by-fee anyway.
I think it would be a mistake to exclude coins that are eligible for de-duplication from the superset rule.
unless I'm missing something (in which case this comment should probably describe the attack in greater details)